rust/matches/src/lib.rs

```diff
@@ -1048,6 +1056,27 @@ pub struct Match {
     pub join_allowed_during_start: bool,
 }
 
+impl Match {
+    pub fn space() -> usize {
```

Comment:
With the number of properties in this struct, keeping the comments would be helpful.